File: 17th Mar 2022/q1.cpp
// 1. Missing number in array

// { Driver Code Starts
// Initial template for C++

#include <bits/stdc++.h>
using namespace std;

 // } Driver Code Ends
// User function template for C++

class Solution{
  public:
    int MissingNumber(vector<int>& a, int n) 
    {
        int s = (n * (n + 1))/2, sum = 0;
        vector <int> :: iterator i; 
        for(i = a.begin(); i != a.end(); i++)
        {
            sum += (*i);
        }
        return s - sum;
    }
};

// { Driver Code Starts.

int main() {
    int t;
    cin >> t;
    while (t--) {
        int n;
        cin >> n;

        vector<int> array(n - 1);
        for (int i = 0; i < n - 1; ++i) cin >> array[i];
        Solution obj;
        cout << obj.MissingNumber(array, n) << "\n";
    }
    return 0;
}  // } Driver Code Ends

File: April/20th Apr 2022/q1.java
// 1. Leap Year

// { Driver Code Starts
//Initial Template for Java

import java.util.Scanner;

class q1 {
    
 // } Driver Code Ends
//User function Template for Java

public static void utility(int y){
    String isLeap = "False";

    //Your code below
    if(y % 4 == 0 && y % 100 != 0)
        isLeap = "True";
    else if(y % 400 == 0)
        isLeap = "True";
    //Your code above
    
    //The line below prints the output
    System.out.println(isLeap);
}

// { Driver Code Starts.

    public static void main(String[] args)
    {
        Scanner scn = new Scanner(System.in);
        int t = scn.nextInt();
        while(t-- > 0) {
            int y = scn.nextInt();
            utility(y);
        }
        scn.close();
    }
}  // } Driver Code Ends
